streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreBuilderWrapper.java

```diff
@@ -34,6 +35,7 @@
  */
 public class StoreBuilderWrapper implements StoreFactory {
 
+
```

Comment:
nit? typo.

Reply:
yeah, typo. I'll remove this in one of my followup PRs so I can go ahead and merge this guy now